stan/math/prim/fun/sort_asc.hpp

```diff
@@ -18,12 +18,12 @@ namespace math {
  * @return Vector in ascending order.
  * @throw std::domain_error If any of the values are NaN.
  */
-template <typename T>
-inline std::vector<T> sort_asc(std::vector<T> xs) {
-  check_not_nan("sort_asc", "container argument", xs);
-  std::sort(xs.begin(), xs.end());
-  return xs;
-}
+        template <typename T>
+        inline std::vector<T> sort_asc(std::vector<T> xs) {
```

Comment:
Oh yeah set your editor to 2 space indents. There's an autoformatter for stuff going into Stan but it's just more space efficient :D

Reply:
done!